examples/generator/railjson_generator/schema/infra/track_section.py

```diff
@@ -29,6 +29,9 @@ class TrackSection:
     end_coordinates: Optional[Tuple[float, float]] = field(default=None)
     begining_links: List[TrackEndpoint] = field(default_factory=list, repr=False)
     end_links: List[TrackEndpoint] = field(default_factory=list, repr=False)
+    slopes: List[Tuple[float, float, float]] = field(default_factory=list)
+    curves: List[Tuple[float, float, float]] = field(default_factory=list)
+    speed_limits: List[Tuple[float, float, str]] = field(default_factory=list)
```

Comment:
I would have added multiple datastruct such as `RangeElement`, `Slope`, `Curve` and `SpeedLimit`  to stay consistent with the rest of the library.